Cards/CardSet.gd

```diff
@@ -0,0 +1,5 @@
+extends Resource
+class_name CardSet
+
+
+@export var card_set: Array[CardBase] = []
```

Comment:
i generally don't like when the variable has the same name as the class because it can be confusing
`CardSetBase` for the class is probably better